docs/book/content/calibration/earnings.md

```diff
@@ -475,6 +475,40 @@ Significant at the 1 percent level (*p* &lt; 0.01). -->
   -
 ```
 
+## Income at the very top
+
+In addition to lifecycle profiles of the seven percentile groups above, `OG-USA` has calibrations of income at the very top.  This includes breaking out percentiles at fine as the top 0.01% of earners.  The two alternative $\lambda$ vectors are $\lambda_{j}=[0.25, 0.25, 0.2, 0.1, 0.1, 0.09, 0.005, 0.004, 0.001]$ and $\lambda_{j}=[0.25, 0.25, 0.2, 0.1, 0.1, 0.09, 0.005, 0.004, 0.0009, 0.0001]$.
+
+Because we do not have panel data that allow us to observe such top percentile groups, we make the following assumptions in calibrating income at the very top.  First, we assume the shape of the lifecycle profile of these top earners is the same as the top 1% overall.  Second, we 2018 estimates from the methodolgy of {cite}`PikettySaez:2003` to provide factors to scale earnings process we estimate for groups inside the top 1%.[^PS_note]
+
+
+```{list-table} **Incomes at the very top** Source: Piketty and Saez (2003) 2018 estimates.
+:header-rows: 1
+:name: tab:top_incomes
+
+* - Income Percentile Range
+  - Top 1%          | $1,143,687  |  1.000 |
+  - Top 1-0.5%        |  $524,677   |  0.459 |
+  - Top 0.5-0.1%       |  $968,991   |  0.847 |
+  - Top 0.1-0.01%      | $3,103,621  |  2.714 |
+  - Top 0.1%         | $4,937,516  |  4.317 |
+  - Top 0.01%        | $21,442,570 | 18.749 |
+* - $1,143,687
+  - $524,677
+  - $968,991
+  - $3,103,621
+  - $4,937,516
+  - $21,442,570
+* - 1.000
+  - 0.459 
+  - 0.847
+  - 2.714
+  - 4.317
+  - 18.749
+```
+
 [^retroquest_note]: The CPS survey asks retrospective questions about income in the last year and average hours worked per week (and weeks worked) in the last year).  Therefore, these CPS surveys line up with tax years 1991-2009.
 
-[^threshold_note]: This threshold is equivalent to \$50 million of wage income in one year at full time (40 hours per week) of work.
\ No newline at end of file
+[^threshold_note]: This threshold is equivalent to \$50 million of wage income in one year at full time (40 hours per week) of work.
+
+[^PS_note]: These data are available from the website of Emmanuel Saez: (https://eml.berkeley.edu/~saez/)[https://eml.berkeley.edu/~saez/].  We use numbers from Table0, Panel B, "Income excluding realized capital gains."
```

Comment:
@jdebacker . The link here should have the square brackets first following the markdown URL form. However, I went to Saez' website and could not find the reference in this footnote. I think you mean the Piketty and Saez (2003) paper.

Reply:
@rickecon The link to the Excel file is on Saez's website under Piketty and Saez (2003).  The 2018 numbers are not in the paper, but linked to on the webpage (and I didn't want to link to a direct download of the Excel file).